wiki/en/Server-Linux.md

```diff
@@ -54,6 +54,11 @@ Set your Directory (genre) server using the `--directoryserver` (`-e`) option as
 |**Genre Classical/Folk** |`classical.jamulus.io:22524`|
 |**Genre Choral/Barbershop** |`choral.jamulus.io:22724`|
 
+
+### Running a Custom Directory
+
+Set the server to use itself (in most cases this will be `localhost`) as the Directory with the `-e` or `--directoryserver` option. Note also that you may want to set the `--directoryfile` option. See [this explanation](Running-a-Server#server-list-filename). 
```

Comment:
> Set the server to use itself (in most cases this will be `localhost`) as the Directory...

Better to say
"Set the server to run as a Directory by using 'localhost' or '127.0.0.1' (exactly) as the `--directoryserver` (`-e`) option."

There's no "in most cases" about it -- those are literal strings that are tested for.